src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -1,41 +1,70 @@
-import React from 'react';
-import { Button, CardGrid } from '@edx/paragon';
+import React, { useState } from 'react';
+import { Button, CardGrid, Toast } from '@edx/paragon';
 import LMSCard from './LMSCard';
+import LMSConfigPage from './LMSConfigPage';
+import {
+  BLACKBOARD_TYPE, CANVAS_TYPE, CORNERSTONE_TYPE, DEGREED_TYPE, MOODLE_TYPE, SAP_TYPE,
+} from '../data/constants';
 
-const SettingsLMSTab = () => (
-  <div>
-    <div className="d-flex">
-      <h2 className="py-2">Learning Management System Configuration</h2>
-      <Button
-        href="https://business-support.edx.org/hc/en-us/categories/360000368453-Integrations"
-        className="ml-auto my-2"
-        rel="noopener noreferrer"
-        target="_blank"
-        variant="outline-primary"
+export default function SettingsLMSTab() {
+  const [config, setConfig] = useState();
+  const [show, setShow] = useState(false);
+
+  const configure = (LMStype) => {
+    if (!LMStype) {
+      setShow(true);
+    }
+    setConfig(LMStype);
+  };
+
+  return (
+    <div>
+      <div className="d-flex">
+        <h2 className="py-2">Learning Management System Configuration</h2>
+        <Button
+          href="https://business-support.edx.org/hc/en-us/categories/360000368453-Integrations"
+          className="ml-auto my-2"
+          rel="noopener noreferrer"
+          target="_blank"
+          variant="outline-primary"
+        >
+          Need Help?
+        </Button>
+      </div>
+      <p>
+        Enabling a learning management system for your edX account allows quick
+        access to the catalog
+      </p>
+      {!config && (
+        <span>
+          <h4 className="mt-4.5 mb-3">New</h4>
+          <CardGrid
```

Comment:
Now that we have a more finalized figma mock, let's try and fit the spacing between cards more closely to what UX envisions. 
![image](https://user-images.githubusercontent.com/67655836/153274009-e112d2fd-70b8-4df6-8f0f-6657e3d08a4f.png)